[CI/Tests] VendorTest.test_create_vendor_with_tags fails due to missing view_tag permission

### Plugin version

v1.1.9

### NetBox version

v4.6.7

### Python version

3.8

### Steps to Reproduce

1. Run the plugin unit test suite:
`python netbox/netbox/manage.py test netbox_lifecycle`
2. Observe the test result for `VendorTest.test_create_vendor_with_tags`.


### Expected Behavior

All unit tests pass successfully in CI.


### Observed Behavior

`VendorTest.test_create_vendor_with_tags` fails with HTTP 400 Bad Request:
```
AssertionError: 400 != 201 : Expected HTTP status 201; received 400: 
{'tags': [ErrorDetail(string="Related object not found using the provided attributes: {'name': 'Test Tag'}", code='invalid')]}
```

Upstream NetBox core commit netbox-community/netbox@e713b4f enforced view permissions when referencing related objects by attributes in nested REST API serializers.
In test_create_vendor_with_tags, self.add_permissions('netbox_lifecycle.add_vendor') is called, but it lacks 'extras.view_tag'.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -55,7 +55,7 @@ jobs:
       uses: actions/checkout@v4
       with:
         repository: 'netbox-community/netbox'
-        ref: 'main'
+        ref: 'v4.6.7'
         path: 'netbox'
 
 

--- a/netbox_lifecycle/tests/test_api.py
+++ b/netbox_lifecycle/tests/test_api.py
@@ -51,7 +51,7 @@ class VendorTest(APIViewTestCases.APIViewTestCase):
 
     def test_create_vendor_with_tags(self):
         """Test creating a vendor with tags via the API."""
-        self.add_permissions('netbox_lifecycle.add_vendor')
+        self.add_permissions('netbox_lifecycle.add_vendor', 'extras.view_tag')
         Tag.objects.create(name='Test Tag', slug='test-tag')
         url = reverse('plugins-api:netbox_lifecycle-api:vendor-list')
         data = {

